azure-pipelines/azure-templates/webhooks-template.yml

```diff
@@ -0,0 +1,31 @@
+parameters:
+  - name: 'projectName'
+    value: string
+  - name: 'channelID'
+    value: string
+  - name: 'webhookKey'
+    value: string
+
+jobs:
+  - job: DiscordMessage
+
+    steps: 
+    - task: ado-discord-webhook@1
```

Comment:
inconsistency in yaml indentation (not blocking)

Reply:
Last push should've solved this.